src/software/ai/hl/stp/play/enemy_free_kick_play.cpp

```diff
@@ -91,13 +91,15 @@ void EnemyFreekickPlay::getNextTactics(TacticCoroutine::push_type &yield,
                     Vector(0, 2 * ROBOT_MAX_RADIUS_METERS),
                 (world_ptr->ball().position() - world_ptr->field().friendlyGoalCenter())
                     .orientation(),
-                0);
+                0, TbotsProto::MaxAllowedSpeedMode::STOP_COMMAND,
```

Comment:
this is both modifying `move_tactic_main`. I think one of them is supposed to be `move_tactic_secondary`

Reply:
Oh wow good call. Seems like this bug's been there for 5+ years :sweat_smile: